backend/research_groups/serializers.py

```diff
@@ -1,17 +1,29 @@
 from rest_framework import serializers
 
 from backend.research_groups.models import ResearchGroup
+from django.contrib.auth.models import User
+
 
 class ResearchGroupSerializer(serializers.ModelSerializer):
-    category = serializers.SerializerMethodField()
+    members = serializers.SlugRelatedField(
+        many=True, slug_field="email", queryset=User.objects.all()
+    )
+
     class Meta:
         model = ResearchGroup
         fields = [
             "id",
             "name",
-            "description",
+            "about_us",
             "category",
+            "members",
         ]
```

Comment:
Jak chcesz wziąć wszystkie pola z modelu to możesz napisać chyba `fields = '__all__'`